### DISCUSSION TITLE
Uninstalling the software

### DISCUSSION THREAD

**Discussion by hicklemcpickle** [ChrisTitusTech/winutil#836] [Q&A]
Hello. I installed Chris Titus Tech about 3 days ago and the software is amazing. I can see some major changes but I feel like it isn't for me. Is there a way to uninstall the software or reset the tweaks and turn my laptop to how it was at first? I'm not sure if this is related but I have been experiencing game crashes and don't know whether if it's a hardware issue or if it's something related to CTT.

**Comment by scudo005** [comment]
If you have checked the box, CTT should have created a restore point before making any changes; you can try to reverse your system to that restore point.
